doc/source/technology_showcase_examples/techdemo-15/ex_15-teccalvalhyper.rst

```diff
@@ -1,4 +1,6 @@
-Technology Showcase Example 15: Calibrating and Validating a Hyperelastic Constitutive Model
+.. _sphx_glr_ex_15-teccalvalhyper.rst:
+
+Calibrating and Validating a Hyperelastic Constitutive Model
 ----------------------------------------------------------------------------------------------
```

Comment:
Watch out for the difference in lines here
```suggestion
Calibrating and Validating a Hyperelastic Constitutive Model
------------------------------------------------------------
```

Reply:
Maybe this is what was causing the build process to fail? Let me merge and check...